src/components/Menu/MenuItem/MenuItem.jsx

```diff
@@ -226,6 +232,7 @@ const MenuItem = forwardRef(
         aria-current={isActive}
         onMouseLeave={onMouseLeave}
         onMouseEnter={onMouseEnter}
+        tabIndex={-1}
```

Comment:
there are many manipulation of focus/blur, it's much easier when each menu item is focusable

Reply:
Very small suggestion - `-1` kinda feels like a "magic" number, maybe assign it to a constant with a meaningful name?